repos.json

```diff
@@ -25,6 +25,10 @@
         "ma27": {
             "url": "https://gitlab.com/Ma27/nur-packages"
         },
+        "makefu": {
+            "file": "makefu/5pkgs/nur.nix",
+            "url": "http://cgit.euer.krebsco.de/stockholm"
```

Comment:
Fetching from your gitlab instance would be a bit faster because we can check gitlab's rss feed.

Reply:
Also it would not be too hard to add cgit support also: https://github.com/nix-community/NUR/blob/master/nur/prefetch.py#L40

Reply:
I can also push to a gitlab instance.

btw test says:
```
access to URI 'https://github.com/Pistos/diakonos' is forbidden in restricted mode
Failed to updated repository manveru
```
hope this is fine